src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/ccd/message/EventInformation.java

```diff
@@ -101,4 +101,15 @@ public String getUserId() {
     public AdditionalData getAdditionalData() {
         return additionalData;
     }
+
+    public static EventInformation getEventInformation() {
+        return EventInformation
+            .builder()
+            .userId("userId_3")
+            .jurisdictionId("jurisdictionId_3")
+            .caseTypeId("caseTypeId_3")
+            .caseId("caseId_3")
+            .eventTimeStamp(LocalDateTime.now())
+            .build();
+    }
```

Comment:
This looks like test code in amongst prod code?  Please remove.

Reply:
I believe this was meant to go [here](https://github.com/hmcts/wa-case-event-handler/blob/5511db89530eed78e4c2fcf993346782ca1dcc34/src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/ccd/message/EventInformationTest.java#L26) but this class is currently broken, so i've moved it back to `ResetNullEventTimestampMessageJobTest`